Products/LDAPUserFolder/tests/test_utils.py

```diff
@@ -51,6 +51,8 @@ def test_createLDAPPassword_crypt(self):
             self.assertTrue(AuthEncoding.pw_validate(encoded, self.pwd))
         except ImportError:
             pass
+        except ValueError:
+            pass
```

Comment:
You cannot have two except clauses following one try. This smells like a syntax error to me.

Reply:
Actually you can, [see an example here](https://pythonbasics.org/try-except/). This is required if handling different exceptions requirers different code. Not sure yet what sense this makes here. -> investigating

Reply:
My mistake, sorry, that was a brain fart.